### PR TITLE
GitHub Super Linter Migration, Lint Fixes, Pipeline cleanup

### PR DESCRIPTION
**Pull request by laughingbiscuit**
What's changed, or what was fixed?

- migrated to github superlinter, increasing lint coverage and reducing pipeline LoC
- many fixes after linting dockerfiles, shell, markdown, java
- improved org cleanup to also cleanup developers, companies, apps
- added more commands to aac 

- [ ] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Review by seymen — APPROVED** [review]

File: .github/workflows/devrel-single.yml [hunk 1]
```diff
@@ -13,80 +13,73 @@
...
-          pipeline-action: check-markdown.sh
-          project: ${{ steps.list-changed.outputs.project }}
+          pipeline-action: list-changed-projects.sh
 
-      - name: check-readme
-        uses: ./tools/pipeline-runner
+      - name: Print Current Project
+        run: echo "${{ steps.list-changed.outputs.project }}"
+      
+      - name: Turnstyle
```
Comment (danistrebel) [inline .github/workflows/devrel-single.yml:71]: Add some explanation? Maybe change name to "Turnstyle - No parallelism in Github workflows"

File: .markdown-lint.yml [hunk 2]
```diff
@@ -13,8 +12,8 @@
...
-
-set -e
-DIR="${1:-$PWD}"
-
-java -jar /opt/java-checkstyle.jar -c "java-checkstyle.xml" "$DIR"
+MD013:
+  code_blocks: false
+MD024:
+  siblings_only: true
+MD041: false
```
Comment (danistrebel) [inline .markdown-lint.yml:19]: Add # First line in file should be a top level heading

File: .markdown-lint.yml [hunk 3]
```diff
@@ -13,8 +12,8 @@
...
 # limitations under the License.
 
-
-set -e
-DIR="${1:-$PWD}"
-
-java -jar /opt/java-checkstyle.jar -c "java-checkstyle.xml" "$DIR"
+MD013:
+  code_blocks: false
+MD024:
```
Comment (danistrebel) [inline .markdown-lint.yml:17]: Add # Multiple headings with the same content

File: .markdown-lint.yml [hunk 4]
```diff
@@ -13,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-set -e
-DIR="${1:-$PWD}"
-
-java -jar /opt/java-checkstyle.jar -c "java-checkstyle.xml" "$DIR"
+MD013:
```
Comment (danistrebel) [inline .markdown-lint.yml:15]: add # Line length

**Review by danistrebel — APPROVED** [review]
LGTM

**Pull request merged**